### PR TITLE
added balance, deposit, and withdraw to `account.py`,  init.py to mod…

### PR DESCRIPTION
**Pull request by hita03**
added balance, deposit, and withdraw to `account.py`,  init.py to modules, `accounts_test` in `tests`
todo - update(changes in account balance) and append(for ministatement) on `database.json`

### CODE REVIEW

File: tests/account_tests.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import unittest
+import requests
+from atmsimulator.account import *
+from unittest.mock import patch
+
+class TestAccount(unittest.TestCase):
+    def setUp(self) -> None:
+        from atmsimulator import globals
+        globals.account={
```
Comment (achyuta116) [inline tests/account_tests.py:9]: `globals.account` within `globals.py` isn't an array of accounts, it's only the account object that is retrieved from the server. Fix your tests and functions within `account.py` accordingly. Apart from that, if the tests work as they're expected to should be all good.

File: tests/database.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
```
Comment (achyuta116) [inline tests/database.json:1]: Replace the contents of this with the database.json file that has more accounts to play around with please. Remember to keep pulling from the main branch to see any changes.

File: atmsimulator/account.py [hunk 3]
```diff
@@ -1,55 +1,122 @@
...
+    elif selection == 'Current':
+        deposit = int(input('Enter amount in Rs. to be deposited: '))
+        globals.account['users'][0]['accounts'][1]['balance']+=deposit
+        print('Deposit successful. Your current balance is: ',globals.account['users'][0]['accounts'][1]['balance'])  
+    else:
+        clear()
+        unsuccessful_transaction_message()
 
 
 def customer_account_mini_statement():
```
Comment (achyuta116) [inline atmsimulator/account.py:92]: I believe @nkilm had written an implementation of mini statement, did you check with him before you made changes?

**Review by achyuta116 — CHANGES_REQUESTED** [review]
Please remove the empty __init__.py file from utils. We only need atmsimulator and tests to be packages, revise imports accordingly since it might break changes already pushed to the main branch.

**Comment by nkilm** [comment]
@hita03 Please use `inquirer` wherever we are taking user input, also add basic verification. E.g, for withdrawl/deposit the input should only be float/int.

**Comment by hita03** [comment]
Got it.

**Comment by hita03** [comment]
* formatted `account.py`. `autopep8` installed for formatting code. It auto-installs other packages.`pip install -U autopep8` only.
* `__init__.py` was added to `utils` folder to modularise it. (for imports)
* `globals.account` in `tests/account_tests.py` has been updated to an object.
* `tests/database.json` was a replica of the actual database. It isn't used in tests as we're only hardcoding `globals.account`. So it's been deleted.
* addition to @nkilm's the mini-statement is a single `if-else` to check whether this service has been disabled, continuing only if it is enabled.
* cash limit of 2,50,000 rs has been included in account.py's `customer_account_cash_withdrawal`
*  fixed up `account.py` - inquirer for all inputs, verification.

**Review by achyuta116 — APPROVED** [review]

**Pull request merged**